### ISSUE TITLE
Iron icon won't disappear if assigned with ""

### ISSUE THREAD

**Issue by dfghi** [PolymerElements/iron-icon#67]
### Description

If an <iron-icon> element has no icon assigned when attached, it stays empty without error. When you assign an icon it displays it, but when you un-assign the icon it keeps the last assigned icon forever.
Example:
https://jsfiddle.net/sasntsjf/2/
### Expected outcome

When the icon property is assigned to null or "", the icon should disappear.
### Actual outcome

When the icon property is assigned to null or "", the last assigned icon is still displayed.
### Live Demo

https://jsfiddle.net/sasntsjf/2/
### Steps to reproduce

Assign a value to the icon property
Assign "" to the icon property
### Browsers Affected

All, it's a problem of behaviour.

**Comment by freshp86** [comment]
This is affecting Chrome's new settings page, [see bug](https://bugs.chromium.org/p/chromium/issues/detail?id=628071).

/cc @danbeam @tjsavage

**Comment by freshp86** [comment]
FYI, I have a candidate fix for this. The problem seems to be caused by these lines https://github.com/PolymerElements/iron-iconset-svg/blob/master/iron-iconset-svg.html#L102-L127

At line 110 the `_svgIcon` property is set on the root. But the equivalent code that removes the `_svgIcon` at line 123 is checking the wrong element. There should be a line

```
element = element.root || element; // This line is missing
if (element._svgIcon) {
  Polymer.dom(element).removeChild(element._svgIcon);
  element._svgIcon = null;
}
```

In other words removeIcon checks the wrong HTML element and the check always fails. I verified that this fixes the issue for Chrome's settings.

**Comment by freshp86** [comment]
And a bug about this already exists at the iron-iconset-svg repository, and a PR , see 
https://github.com/PolymerElements/iron-iconset-svg/issues/45 and 
https://github.com/PolymerElements/iron-iconset-svg/pull/44.

**Comment by bicknellr** [comment]
This was fixed by #44.
